working/augmentation-libraries/feature-specification.md

```diff
@@ -377,36 +378,39 @@ including enum values, are added to the instance or static namespace of the
 corresponding type in the augmented library. In other words, the augmentation
 can add new members to an existing type.
 
-Instance and static members inside a type may themselves be augmentations. In
-that case, they augment the corresponding members in augmented type
-declaration (a based declaration and zero or more augmentations that are all
-above the current augmenting type declaration) according to the rules in the
-following subsections.
+Instance and static members inside a class-like declaration may themselves be
+augmentations. In that case, they augment the corresponding members in the
+augmented class-like declaration (a based declaration and zero or more
+augmentations that are all above the current augmenting type declaration)
+according to the rules in the following subsections.
+
+It's a **compile-time** error if a library contains two top-level declarations
+with the same name, and:
+
+*   Neither is an augmenting declaration, or
+*   one of the declarations is a class-like declarations and the other is not of
+    the same kind, meaning that at either one is a class, mixin, enum, extension
+    or extension type declaration, and the other is not the same kind of
+    declaration.
```

Comment:
Isn't this error the same as the error below?

Reply:
Yup, just forgot to delete that one when moving it. I will just push a fix directly.